examples/neural_compressor/text-to-image/README.md

```diff
@@ -0,0 +1,51 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# text to image 
+
+The script [`run_diffusion.py`](https://github.com/huggingface/optimum-intel/blob/main/examples/neural_compressor/text-to-image/run_diffusion.py)
+allows us to apply different quantization approaches (such as dynamic, static) using the [Intel Neural Compressor ](https://github.com/intel/neural-compressor) library for 
+text to image tasks.
+
+The following example applies post-training static quantization on a stable-diffusion with pretrained model:[CompVis/stable-diffusion-v1-4](https://huggingface.co/CompVis/stable-diffusion-v1-4). In this example, we only quantized the unet model in the diffusion pipeline, and the diffusion pipeline has four models: safety_checker, text_encoder, unet, vae. 
```

Comment:
only quantize the unet model which is the performance critical in the diffusion pipeline